docs/source/model-registry.rst

```diff
@@ -35,6 +35,9 @@ Annotations and Descriptions
 
 Model Registry Workflows
 ========================
+In order to use model registry functionality via the UI or API, ensure you're running
```

Comment:
Would welcome feedback on the wording here, it may confuse users of managed MLflow servers (i.e. users who aren't running their own MLflow server)